OsmAnd/src/net/osmand/plus/onlinerouting/OnlineRoutingHelper.java

```diff
@@ -122,10 +123,16 @@ public String makeRequest(@NonNull String url) throws IOException {
 			return makeRequest(url, "GET", null, null);
 	}
 
+	private final Map<String, String> simpleHttpCache = new ConcurrentHashMap<>();
```

Comment:
Such cache looks strange

Reply:
that's the simplest method to avoid double-http-requests